CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 1.5.0
+
+ABQ 1.5.0 adds new functionality for test distribution and the ability to replay
+tests locally without modifying the remote queue.
+
+By default, ABQ assigns the next available test to a worker that requests tests.
```

Comment:
I don't think we should block on this but we might want to add links to the docs here later.